app/src/main/res/values/fractions.xml

```diff
@@ -1,4 +1,4 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-	<fraction name="hero_aspect_ratio">178%</fraction>
+	<fraction name="hero_aspect_ratio">W,16:9</fraction>
```

Comment:
Reading the documentation, I think this should be H,16:9 - we want to constrain the height by the ratio while the width is constrained by the end/start constraints in the ImageView. We also should change the value in the landscape version of `fractions.xml` to be 32:9